vite.config.ts

```diff
@@ -25,7 +25,12 @@ export default defineConfig(({ mode }) => {
     plugins: [svgr(), preact()],
 
     test: {
-      environment: 'happy-dom',
+      browser: {
```

Comment:
It looks like we can do cross-browser testing with vitest, it'd be good to test with safari and firefox if we can (as well as chrome), so I've raised WRAP-272 to track.